os/board/rtl8721csm/src/component/common/drivers/wlan/realtek/include/autoconf.h

```diff
@@ -507,7 +507,7 @@ extern unsigned int g_ap_sta_num;
 		#undef RX_AGGREGATION
 		#define RX_AGGREGATION 1
 		#undef NOT_SUPPORT_40M
-		#define RX_SHORTCUT 1
+		//#define RX_SHORTCUT 1

```

Comment:
How about removing this line if this is not used anymore?

Reply:
ok can